lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -43,7 +43,15 @@ namespace gpu {
 unsigned getTotalElemsPerThread(Attribute layout, ArrayRef<int64_t> shape,
                                 Type eltTy) {
   if (auto tritonGPUAttr = layout.dyn_cast<TritonGPU_AttrTrait>()) {
-    return tritonGPUAttr.getTotalElemsPerThread(shape, eltTy);
+    unsigned elemNum = tritonGPUAttr.getTotalElemsPerThread(shape, eltTy);
```

Comment:
I think it's better not to touch this "interface" function. The changes are for dotOp with mfma as parent anyway.

Reply:
Thanks. Changed accordingly.